### PR TITLE
Change license to reflect other projects

### PR DESCRIPTION
**Pull request by mmahalwy**
Fixes https://github.com/quran/quran.com-frontend/issues/714

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32827

File: LICENSE.md [hunk 1]
```diff
@@ -1,21 +1,674 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    Quran for iOS is a Quran reading application for iOS.
```
Comment (naveed-ahmad) [inline LICENSE.md]: Ee need to change the wording here.

File: LICENSE.md [hunk 2]
```diff
@@ -1,21 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    Quran-iOS Copyright (C) 2017 Quran.com
```
Comment (naveed-ahmad) [inline LICENSE.md]: here as well

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32836

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32840

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32841

**Pull request merged**